applications/FluidDynamicsApplication/custom_constitutive/newtonian_two_fluid_2d_law.cpp

```diff
@@ -54,6 +54,24 @@ std::string NewtonianTwoFluid2DLaw::Info() const {
     return "NewtonianTwoFluid2DLaw";
 }
 
+int NewtonianTwoFluid2DLaw::Check(
+    const Properties& rMaterialProperties,
+    const GeometryType& rElementGeometry,
+    const ProcessInfo& rCurrentProcessInfo)
+{
```

Comment:
Please use the new interface aka make the fct const

Reply:
I think this is the current interface of the CL, isnt it?

https://github.com/KratosMultiphysics/Kratos/blob/b2c2c61f73985d1c9702d3999866285ab8ed5a56/kratos/includes/constitutive_law.h#L1164

Reply:
I think so too. The CL has no `const` yet (this happened to me a few weeks ago).

Reply:
Ah sorry I confused it with the new Element/Condition interfaces :D

Sorry for the noise, please go ahead

(we should probably make the `Check` of the `ConstitutiveLaw` `const` too)

Reply:
+1 `const` CL `Check`

Reply:
Agree, I will merge this for the moment if thats ok for you